worlds/rogue_legacy/Regions.py

```diff
@@ -100,24 +95,18 @@ def create_regions(multiworld: MultiWorld, player: int):
     multiworld.get_entrance("The Fountain Room", player).connect(multiworld.get_region("The Fountain Room", player))
 
 
-def create_region(multiworld: MultiWorld, player: int, name: str, locations=None, exits=None):
-    ret = Region(name, RegionType.Generic, name, player)
-    ret.multiworld = multiworld
+def create_region(multiworld: MultiWorld, player: int, name: str, locations=None, region_exits=None):
+    region = Region(name, RegionType.Generic, name, player)
+    region.multiworld = multiworld
     if locations:
         for loc_name in locations:
             loc_data = location_table.get(loc_name)
-            location = RLLocation(player, loc_name, loc_data.code if loc_data else None, ret)
-
-            # Special rule handling for fairy chests.
-            if "Fairy" in loc_name:
-                location.access_rule = lambda state: state.has("Dragons", player) or (
-                        state.has("Enchantress", player) and (
-                        state.has("Vault Runes", player) or
-                        state.has("Sprint Runes", player) or
-                        state.has("Sky Runes", player)))
-            ret.locations.append(location)
-    if exits:
-        for exit in exits:
-            entrance = Entrance(player, exit, ret)
-            ret.exits.append(entrance)
-    return ret
+            location = RLLocation(player, loc_name, loc_data.code if loc_data else None, region)
```

Comment:
Since you're already defaulting this to None this doesn't need to be an if else?

Reply:
`loc_data.code` will throw if `loc_data` is None.